deployment/network-operator/templates/mellanox.com_v1alpha1_nicclusterpolicy_cr.yaml

```diff
@@ -18,8 +18,6 @@ apiVersion: mellanox.com/v1alpha1
 kind: NicClusterPolicy
 metadata:
   name: nic-cluster-policy
-  annotations:
-    helm.sh/hook: post-install,post-upgrade
```

Comment:
Idea (needs to be checked):

introduce scale down of netop deployment on post upgrade. deploy CR as hook with higher weight, scale back up as hook with even higher weight 

on post install we just deploy CR 

on post delete we delete CR

use  helm.sh/resource-policy: keep  -> should not delete existing resource (i hope it will do resouce update)

migration logic - remove the label from config map to prevent operator from removing the config map


with this i believe we can keep admission controller logic.

Reply:
Update:

after internal discussion and testing with @rollandf this will not work.